pkg/config/config.go

```diff
@@ -47,6 +47,8 @@ type Configuration struct {
 
 	Docker   *Docker   `yaml:"docker,omitempty"`
 	Deployer *Deployer `yaml:"deployer,omitempty"`
+
+	NoRC bool `yaml:"noreleasecanditates,omitempty"`
```

Comment:
what abount a 
```
CustomBumpStrategies map[string]string `yaml:"custom_bump_strategies"`
```

with value like:
```
custom_bump_strategies:
   staging: bump_patch
```
in `pkg/bumper/bumper.go` we can add a global variable like `var bumpStrategies = map[string]func(*version.Version){ ... }`